packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -165,6 +165,9 @@ export class ProfilesCache {
         let mProfileInfo: zowe.imperative.ProfileInfo;
         try {
             mProfileInfo = await this.getProfileInfo();
+            if (!mProfileInfo.usingTeamConfig) {
+                return;
+            }
```

Comment:
I don't think this will conflict with how the ProfileInfo supports v1 vs v2 profiles.
However, this may be unnecessary once the ProfileInfo removes v1 support 😋

Reply:
for now this returns the empty arrays if using v1 profiles or user has no profiles, but yes once implemented on cli we shouldn't need anymore